src/clients/mapproxyConfig.ts

```diff
@@ -30,17 +30,13 @@ export class MapproxyConfigClient extends HttpClient {
   @withSpanAsyncV4
   public async getConfig(format: SchemaType = SchemaType.YAML): Promise<IMapProxyConfig | Promise<string>> {
     const spanActive = trace.getActiveSpan();
-    spanActive?.setAttributes({
-      /* eslint-disable @typescript-eslint/naming-convention */
-      'mapcolonies.raster.mapproxyApiUrl': this.getConfigUrl,
-      /* eslint-enable @typescript-eslint/naming-convention */
-    });
     // eslint-disable-next-line @typescript-eslint/naming-convention
     const headers = { Accept: format };
     try {
+      const mapproxyApiUrl = join([this.config.get<string>('mapproxy.mapproxyApiUrl'), this.getConfigUrl], '');
```

Comment:
why do you need the join method to be activated every time this method is called and not just once in constructor?

Reply:
it's for logging, the actual concat occurs inside the HTTP client class (hidden from the developer)

Reply:
as discussed

Reply:
done